LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Least Authority TFA GmbH
```

Comment:
Earliest commit is in 2021, your discretion on whether to reflect that here or if you have another reason for the year

Reply:
I guess it just feels like it's been longer :sweat_smile: - will fix